relation_engine_server/main.py

```diff
@@ -43,8 +43,8 @@ def return_error(error_dict, code):
 @app.route("/", methods=["GET"])
 def root():
     """Server status."""
-    if os.path.exists(".git/refs/heads/master"):
-        with open(".git/refs/heads/master", "r") as fd:
+    if os.path.exists(".git/refs/heads/develop"):
```

Comment:
So you can get the right commit version when `curl`ing instead of just getting "unknown" commit. Note to self: document this in later PR

Reply:
Can you just add a comment in the code here?